dtfs-central-api/package.json

```diff
@@ -46,7 +46,7 @@
     "lodash": "^4.17.21",
     "lodash.orderby": "^4.6.0",
     "mongo-dot-notation": "3.1.0",
```

Comment:
does this need ^ as well?

Reply:
Yes good shout, I'll add this in https://github.com/UK-Export-Finance/dtfs2/pull/3598 PR, is that okay?